src/services/types.ts

```diff
@@ -0,0 +1,21 @@
+import { Json, u32 } from "@polkadot/types";
+import React from "react";
+
+export interface SchemaDetails {
```

Comment:
I find these are usually put in the same file as the component, but I might be out of date on my react patterns.

Reply:
I'm also using SchemaDetails in extrinsic.ts so figured it was best to leave it separate?

Reply:
Ah that's fine then.